File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        res = [0,0]
        indexes = {}
        i = 0
        for number in nums:
            check = indexes.get(number,-1)
            if check != -1:
                res = [i,indexes[number]]
                return res
            indexes[target-number] = i
            i=i+1
        return res
    

File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer,Integer> map = new HashMap<>();
        for(int i = 0;i<nums.length;i++){
            if(map.get(nums[i])==null){
                map.put(target-nums[i], i);
            }else{
                int[] a = {i,map.get(nums[i])};
                return a;
            }
        }
        return new int[2];
    }
}